lambdas/validate-oauth-callback/src/main/java/uk/gov/di/ipv/core/validateoauthcallback/ValidateOAuthCallbackHandler.java

```diff
@@ -86,37 +86,37 @@ public ValidateOAuthCallbackHandler() {
     @Override
     @Tracing
     @Logging(clearState = true)
-    public Map<String, Object> handleRequest(CredentialIssuerRequestDto request, Context context) {
```

Comment:
Param name can be criCallbackRequest instead of callbackRequest.

Reply:
I think I'm happy with `callbackRequest` as the var name. It's a reasonable balance between verbosity and conciseness in this context.